azkaban-common/src/main/java/azkaban/executor/container/ContainerizedDispatchManager.java

```diff
@@ -149,15 +149,15 @@ public List<Integer> getRunningFlowIds() {
   public List<Integer> getQueuedFlowIds() {
     final List<Integer> allIds = new ArrayList<>();
     try {
-      getExecutionIdsHelper(allIds, this.executorLoader.fetchQueuedFlows(Status.READY));
+      getExecutionIdsHelper(allIds, this.executorLoader.fetchQueuedFlows(Status.PREPARING));
```

Comment:
I think DISPATCHING should also be included in getQueuedFlowIds function.

Reply:
Added dispatching.